rasa/core/utils.py

```diff
@@ -477,7 +477,7 @@ def replace_floats_with_decimals(obj: Union[List, Dict]) -> Any:
             obj[j] = replace_floats_with_decimals(obj[j])
         return obj
     elif isinstance(obj, float):
-        return Decimal(obj)
+        return round(Decimal(obj), 9)
```

Comment:
7 should even be enough (I just checked how many decimals `time.time()` returns

Reply:
This function will also consume confidence scores, hence the slightly higher value:

```
{'event': 'user', 'timestamp': Decimal('1579507733.1107571125030517578125'), 'text': 'hi', 'parse_data': {'intent': {'name': 'greet', 'confidence': Decimal('0.918040672051180450807805755175650119781494140625')}
```